src/Sulu/Bundle/AdminBundle/Resources/js/stores/userStore/tests/updateRouterAttributesFromUserStoreContentLocale.test.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+import updateRouterAttributesFromUserStoreContentLocale from '../updateRouterAttributesFromUserStoreContentLocale';
+import Route from '../../../services/Router/Route';
+
+let mockedContentLocale = undefined;
+
+jest.mock('../../../stores/userStore/userStore', () => ({
+    get contentLocale() {
+        return mockedContentLocale;
```

Comment:
Did the old way stop working? If it is just about flow, which rightfully complains that you can't assign something to a getter, we used the // $FlowFixMe command to get rid of these warnings, since they don't apply here.